desktop/mac/release-notes/index.md

```diff
@@ -33,6 +33,41 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 > chip](https://desktop.docker.com/mac/main/arm64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-arm64){:
 > .button .primary-btn }
 
+### New
+
+- Released Docker Desktop for Linux, supporting deb and rpm packaging formats.
+- Beta release of Docker Extensions and Extensions SDK.
+- [Compose V2 has been marked GA](https://www.docker.com/blog/announcing-compose-v2-general-availability/) 
+### Upgrades
+- [Go 1.18.1](https://golang.org/doc/go1.18)
+
+### Bug fixes and minor changes
```

Comment:
We need for-mac bug numbers on all of these, where they exist.

Reply:
Done! Only 2 is what I've found.